apps/web/src/api/routes/v1_event.ts

```diff
@@ -1,14 +1,31 @@
 import { zValidator } from "@hono/zod-validator";
 import { abbyEventSchema, AbbyEventType } from "@tryabby/core";
-import { Ratelimit } from "@upstash/ratelimit";
-import { Redis } from "@upstash/redis";
 import { Hono } from "hono";
 
 import isbot from "isbot";
 import { EventService } from "server/services/EventService";
 import { RequestCache } from "server/services/RequestCache";
 import { RequestService } from "server/services/RequestService";
 
+import { RateLimiterRedis } from "rate-limiter-flexible";
+import { redis } from "server/db/redis";
+
+export const rateLimiter = new RateLimiterRedis({
+  storeClient: redis,
+  keyPrefix: "rateLimiter",
+  points: 10, // Number of points
+  duration: 10, // Per 10 seconds
+});
+
+export const checkRateLimit = async (ip: string) => {
```

Comment:
I don't think this should be exported here :)

Reply:
Will make changes accordingly and update you.